src/main/resources/application.conf

```diff
@@ -18,15 +18,19 @@ lorre: {
 }
 
 # Used for configuring Typesafe Slick. Replace database name, user and password in an env-specific config file.
-conseildb = {
-  dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
-  properties = {
-    databaseName = "conseil"
-    user = "foo"
-    password = "bar"
-    reWriteBatchedInserts = true
+db {
```

Comment:
Could we change the nesting levels so it's like this?

```scala
databases = {
	tezos {
		zeronet = {
			conseildb = {
					dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
				    properties = {
				    databaseName = "conseil"
				    user = "foo"
				    password = "bar"
				    reWriteBatchedInserts = true
			    }
			  numThreads = 10
			}
		}
	}
}
```

What do you think, @ba205 and @anonymoussprocket ?

Reply:
I've changed it the way you suggested